### ISSUE TITLE
Add URL Validator 

### ISSUE THREAD

**Issue by ottonomy** [openbadges/badgecheck#44]
```py
from badgecheck import tasks
tasks.validation.PrimitiveValueValidator(tasks.validation.ValueTypes.URL)("http://example.com/1")
```

* Should enforce parsability (watch out for differences between py2/py3 in urllib libraries
* Should enforce either HTTP or HTTPS scheme

**Comment by ViktorHaag** [comment]
Implemented with PR  #55

**Comment by mgylling** [comment]
Done in #55.
